packages/app-store/apps.metadata.generated.ts

```diff
@@ -21,6 +21,7 @@ import ga4_config_json from "./ga4/config.json";
 import { metadata as giphy__metadata_ts } from "./giphy/_metadata";
 import { metadata as googlecalendar__metadata_ts } from "./googlecalendar/_metadata";
 import { metadata as googlevideo__metadata_ts } from "./googlevideo/_metadata";
+import greet_meet_config_json from "./greet-meet/config.json";
```

Comment:
This looks like an unrelated change

Reply:
Yes, I'll remove it. I was playing around with the app creator and the changes persisted across branches.